Argo/Globals/JSON.swift

```diff
@@ -15,9 +15,10 @@ public extension JSON {
     case let v as [AnyObject]: return .Array(v.map(parse))
 
     case let v as [Swift.String: AnyObject]:
-      return .Object(reduce(v.keys, [:]) { accum, key in
-        let parsedValue = (self.parse <^> v[key]) ?? .Null
-        return accum + [key: parsedValue]
+      return .Object(reduce(v, [:]) { (var accum, elem) in
+        let parsedValue = (self.parse <^> elem.1) ?? .Null
+        accum[elem.0] = parsedValue
```

Comment:
But `accum` is a value object, so it's passed by copy. It's only mutating the instance in the local scope.